### DISCUSSION TITLE
Airbyte Slowness during Syncing source & destination

### DISCUSSION THREAD

**Discussion by ChidambaramCG** [airbytehq/airbyte#29201] [General]
Hi Team,

We are using Airbyte OSS in our AWS EC2 Instance of size t2.2X large and we are facing delays in getting the tables listed in our snowflake destination more frequently. Issue started a week back and we are using Airbyte OSS for almost 16 months. We are currently on version 0.44.5.

On looking at the logs in Airbyte UI, We could see the Airbyte temporal job got invoked slowly and also the normalization process started slowly after temporal job invocation.

We are using the free trial for most sources and therefore it wont have large volumes of data. Most of them will be of Kilobyte sized data. Attached is the log for reference.

Kindly advise. Looking forward for your reply.

Thanks
Chidambara Ganapathy
[Asana connector log.txt](https://github.com/airbytehq/airbyte/files/12288127/Asana.connector.log.txt)

**Comment by marcosmarxm** [comment]
How many connections you have currently in your instance? Can you check/monitor the cpu/memory for each service?

[a comment by an automated account is not shown]
